be/src/exec/vectorized/aggregator.h

```diff
@@ -151,7 +151,7 @@ class Aggregator final : public pipeline::ContextWithDependency {
     bool _is_closed = false;
     RuntimeState* _state = nullptr;
 
-    const TPlanNode& _tnode;
+    const TPlanNode _tnode;
```

Comment:
const & is just ok, why remove &

Reply:
The _tnode is only valid in the init and prepare phases and will be destructured after the prepare phase